#include "team_client_player_handler.h"
#include "thread_local/storage.h"
#include "game_logic/network/message_util.h"
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
void ClientPlayerTeamServiceHandler::TeamInfoNotify(entt::entity player,
	const ::TeamInfoS2CRequest* request,
	::TeamInfoS2CResponse* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}

